app/views/unattended/provisioning_templates/registration/remote_execution_pull_setup.erb

```diff
@@ -21,7 +21,7 @@ set -e
 
 echo "Getting configuration from subscription-manager..."
 KPPTEMPFILE=$(mktemp kpp_tempfile_XXXXXXXX)
-trap "rm -f $KPPTEMPFILE" ERR EXIT
+trap "rm -f $KPPTEMPFILE" EXIT
```

Comment:
This was taken from https://github.com/theforeman/katello-pull-transport-migrate/commit/c75a9df459881818d86bd24a236edb5052282306#diff-e66411028717f9cf79ffe6f57f6972d581c0106efb104a8596848cbdea42a741R14 . @ekohl do you recall if there was a reason for trying to trap both ERR and EXIT?

Reply:
It's a good question. I think it's mostly something I got used to in my scripts without digging in deeper. According to `man trap` the `ERR` trap is executed if a command returns a non-zero exit code. That does seem redundant if you already use `set -e` (which is also non-POSIX btw) to exit on errors and then trap `EXIT`.